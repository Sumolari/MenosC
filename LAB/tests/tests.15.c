{
	int i;
	i = 99;
	bool logico;
	logico = i > 5;
}