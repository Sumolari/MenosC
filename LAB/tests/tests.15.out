  0 EASIG     i:  99  ,          ,  p:   1  
  1 EASIG     p:   1  ,          ,  p:   0  
  2 EASIG     p:   0  ,          ,  p:   3  
  3 EASIG     i:   5  ,          ,  p:   4  
  4 EASIG     i:   0  ,          ,  p:   5  
  5 EMENEQ    p:   3  ,  p:   4  ,  e:   7  
  6 EASIG     i:   1  ,          ,  p:   5  
  7 EASIG     p:   5  ,          ,  p:   2  
  8 FIN               ,          ,          
